recOrder/cli/apply_inverse_transfer_function.py

```diff
@@ -280,17 +233,15 @@ def apply_inverse_transfer_function_cli(
                 reconstructed_parameters_2d = inplane_oriented_thick_pol3d.apply_inverse_transfer_function(
                     tczyx_data[time_index],
                     intensity_to_stokes_matrix,
-                    wavelength,
-                    cyx_no_sample_data,
+                    cyx_no_sample_data=cyx_no_sample_data,
                     project_stokes_to_2d=True,
                     **biref_inverse_dict,
                 )
 
                 reconstructed_parameters_3d = inplane_oriented_thick_pol3d.apply_inverse_transfer_function(
                     tczyx_data[time_index],
                     intensity_to_stokes_matrix,
-                    wavelength,
-                    cyx_no_sample_data,
+                    cyx_no_sample_data=cyx_no_sample_data,
                     project_stokes_to_2d=False,
```

Comment:
should this be consistent to previous?
(ndim==2)

Reply:
Sorry, I'm having trouble figuring out what you mean here. Can you expand on this comment and the next?

Reply:
This block can only be reached `recon_dim == 3` (similar for `recon_dim == 2`), so I think I'm already using `recon_dim` in the way you're suggesting. 

The birefringence + phase reconstructions in 2D and 3D have relatively intricate logic wrt the waveorder calls, so I don't think there's a dramatic simplification here (I'm open to being wrong though!).